framework-cloud/framework-cloud-api/src/main/java/org/kie/cloud/api/deployment/Instance.java

```diff
@@ -50,6 +50,13 @@
      */
     boolean isRunning();
 
+    /**
+     * Return true if instance is existing on Openshift.
+     *
+     * @return True if instance is existing on Openshift.
+     */
+    boolean isExisting();
```

Comment:
Nitpicking: why the continuous form, wouldn't be just "exists()" fine?

Reply:
changed